### PR TITLE
Update to version 7.1.5

### PR DESCRIPTION
**Pull request by dtchanpura**
Pull Request for Version 7.1.5

### CODE REVIEW

File: io.dbeaver.DBeaverCommunity.appdata.xml [hunk 1]
```diff
@@ -31,6 +31,12 @@
         </screenshot>
     </screenshots>
     <releases>
+        <release version="7.1.5" date="2020-08-16">
+            <description>
+                <p>Release of version: 7.1.5</p>
+            </description>
+            <url>https://github.com/dbeaver/dbeaver/releases/tag/7.1.5</url>
```
Comment (dac73) [inline io.dbeaver.DBeaverCommunity.appdata.xml:38]: This is not shown on flathub.com
Can you check on how to show url with changes on flathub?

Comment (dtchanpura) [reply io.dbeaver.DBeaverCommunity.appdata.xml:38]: I checked Flathub's [Linux Store Backend](https://github.com/flathub/linux-store-backend) which stores the details about each release, the database schema does not have anything for this URL which is defined in appstream.

After looking at the [code](https://github.com/flathub/linux-store-backend/blob/master/src/main/java/org/flathub/api/model/AppRelease.java) I can say we might just have to use description to have something shown in the UI.

Database Schema Reference: https://github.com/flathub/linux-store-backend/tree/master/src/main/resources/db/migration/V1

Comment (dac73) [reply io.dbeaver.DBeaverCommunity.appdata.xml:38]: Yeah, just add in a description, so people can follow it.

Currently, it sits there empty :/

Comment (dtchanpura) [reply io.dbeaver.DBeaverCommunity.appdata.xml:38]: Yeah, sure, let me add that.

**Pull request merged**